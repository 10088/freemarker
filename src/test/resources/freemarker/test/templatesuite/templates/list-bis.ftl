<#list ['a', 'b', 'c'] as x>
    ${x?index}: ${x} [<#list ['A', 'B', 'C'] as x>${x?index}:${x}<#sep>, </#list>]
</#list>

<#list ['a', 'b', 'c'] as i>
    ${i?index}: ${i} <#list ['A', 'B', 'C'] as j>${i?index}${i}/${j?index}${j}<#sep>, </#list>
</#list>

<#list ['a', 'b', 'c']><#items as x>${x?index}:${x}<#sep>, </#items></#list>
<#list ['a', 'b', 'c']><#items as x>${x?counter}. ${x};<#sep> </#items></#list>

<#list ['a', 'b', 'c'] as x>${x}<#if x?hasNext>, </#if></#list>
<#list ['a', 'b', 'c'] as x\-y>${x\-y}<#if x\-y?hasNext>, </#if></#list>

<#list ['a', 'b', 'c'] as x><#if x?isFirst>${x?capFirst}<#else>${x}</#if><#sep>, </#sep><#if x?isLast>.</#if></#list>