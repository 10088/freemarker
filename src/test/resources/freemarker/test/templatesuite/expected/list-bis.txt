    0: a [0:A, 1:B, 2:C]
    1: b [0:A, 1:B, 2:C]
    2: c [0:A, 1:B, 2:C]

    0: a 0a/0A, 0a/1B, 0a/2C
    1: b 1b/0A, 1b/1B, 1b/2C
    2: c 2c/0A, 2c/1B, 2c/2C

0:a, 1:b, 2:c
1. a; 2. b; 3. c;

a, b, c